src/Organizations/Models/Organizations.php

```diff
@@ -25,4 +26,25 @@ public function initialize()
             new Uuid()
         );
     }
+
+    /**
+     * Before create
+     *
+     * @return void
+     */
+    public function beforeCreate() : void
+    {
+        $this->slug = Slug::generate($this->name);
+        parent::beforeCreate();
+    }
+
+    /**
+     * Before save
+     *
+     * @return void
+     */
+    public function beforeSave() : void
+    {
+        $this->slug = Slug::generate($this->name);
```

Comment:
@arfenis remember slug shoulnt change everytime, its has to be created and than update manually if needed, since this can be used for search o URL params